src/Plugin/Field/FieldType/OgGroupItem.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og\OgGroupItem.
+ */
+
+namespace Drupal\og\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+
+/**
+ * OG Group Data class to return TRUE always.
+ *
+ * @FieldType(
```

Comment:
Awesome. That saves me writing it! I hate thinking of what to write for
things like that :) :+1:
On 27 Nov 2015 20:49, "Amitai Burstein" <notifications@github.com> wrote:

> In src/Plugin/Field/FieldType/OgGroupItem.php
> <https://github.com/amitaibu/og/pull/40#discussion_r46069651>:
>
> > +<?php
> > +
> > +/**
> > + * @file
> > + * Contains \Drupal\og\OgGroupItem.
> > + */
> > +
> > +namespace Drupal\og\Plugin\Field\FieldType;
> > +
> > +use Drupal\Core\Field\FieldItemBase;
> > +use Drupal\Core\Field\FieldStorageDefinitionInterface;
> > +
> > +/**
> > + * OG Group Data class to return TRUE always.
> > + *
> > + * @FieldType(
>
> // This field type is used simply as a placeholder for a field formatter
> that would allow non-groups members, members, and group managers to join,
> request or leave the a group.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/40/files#r46069651>.
>